connaisseur/res/config_schema.json

```diff
@@ -152,11 +152,11 @@
                                         "secret_name": {
                                             "type": "string",
                                             "pattern": "[a-z0-9]([-a-z0-9]*[a-z0-9])?(\\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*"
+                                        },
+                                        "k8s_keychain": {
+                                            "type": "boolean"
```

Comment:
If I am not mistaken, `k8s_keychain` won't work with `secret_name` as providing both instruction in conjunction would be undefined behavior. Thus, the object should be `OneOf` as used above for the `notary.auth`: https://github.com/sse-secure-systems/connaisseur/pull/551/files#diff-71bf3c9c9646a106b18d918f9cbdd6899845b0d55a2c254f8a7d7ee83149375fL74

Reply:
would also maintain the `required` if auth is given :slightly_smiling_face:

Reply:
Done